.env.vagrant

```diff
@@ -34,6 +34,7 @@ OAUTH2_CLIENT_ID=
 OAUTH2_CLIENT_SECRET=
 
 COMPANY_AUTH_VERIFICATION_FEATURE_ENABLED=1
+ENABLED_APPEAL_REASONS=illness,other
```

Comment:
This way feels confusing to me because removing 'other' from the list does not change much cause other reason screens will still be available, they might not be easily reachable from the reason list.

There is also no need to keep making list of reasons configurable, the feature flag is needed only to temporarily disable part of application under development in production until development is complete.

Reply:
This is the most compact solution I could think of for allowing the Choose Reasons page to dynamically enable/disable reasons. While this isn't high-priority for the current journey (because disabling Illnesses should simply disable the entire Choose Reason page), it makes it far easier to enable/disable upcoming reason journeys during their development.

I agree that having "other" in there is unnecessary though. I can include some additional logic so that "other" is included by default.

Reply:
Resolved: Switched to binary toggle